ChaosMod/Effects/EffectsInfo.h

```diff
@@ -506,13 +507,13 @@ const std::unordered_map<EffectType, EffectInfo> g_effectsMap =
 	{EFFECT_PLAYER_KEEP_RUNNING, {"Help My W Key Is Stuck", "player_keeprunning", true}},
 	{EFFECT_VEH_WEAPONS, {"Vehicles Shoot Rockets (Left Click/RB)", "veh_weapons", true}},
 	{EFFECT_MISC_AIRSTRIKE, {"Airstrike Inbound", "misc_airstrike", true}},
-	{EFFECT_FAKE_PLAYER_SWAP, {"Player Swap", "player_playerswap", true, {}, true}},
+	{EFFECT_FAKE_PLAYER_SWAP, {"Player Swap", "player_playerswap", true, {}, true, EffectExecutionType::DEFAULT, "Player Swap"}},
 	{EFFECT_PEDS_MINIONS, {"Minions", "peds_minions", true}},
 	{EFFECT_PEDS_MERCENARIES, {"Mercenaries", "peds_mercenaries", true}},
 	{EFFECT_LOOSE_TRIGGER, {"Loose Triggers", "peds_loosetrigger", true}},
 	{EFFECT_MISC_FLAMETHROWER, {"Flamethrowers", "misc_flamethrower", true}},
 	{EFFECT_MISC_DVDSCREENSAVER, {"DVD Screensaver", "misc_dvdscreensaver", true, {}, true}},
-	{EFFECT_PLAYER_FAKEDEATH, {"Fake Death", "player_fakedeath"}},
+	{EFFECT_PLAYER_FAKEDEATH, {"Fake Death", "player_fakedeath", false, {}, false, EffectExecutionType::DEFAULT, "Suicide"}},
```

Comment:
The issue is this effect doesn't always play the suicide animation (it can also either do no animation or explode the vehicle the player is in). It should rather be specified in the effect directly so it's able to show a fake name corresponding to the action it's doing (`Suicide` if it's playing the suicide animation, `Explode Current Vehicle` if it's faking a vehicle explosion and nothing if no animation is being played).

Also it's preferable to make it use either the original or (if available) custom name of another effect directly instead of using hardcoded texts (maybe by adjusting the `OverrideEffectName` to take another effect enum item and fetch the name based on that instead).